store/sqlstore/supplier.go

```diff
@@ -59,6 +59,8 @@ const (
 	EXIT_REMOVE_INDEX_MYSQL          = 122
 	EXIT_REMOVE_INDEX_MISSING        = 123
 	EXIT_REMOVE_TABLE                = 134
+	EXIT_CREATE_INDEX_SQLITE         = 135
```

Comment:
I copied this pattern blindly, but it wasn't really clear what value we're getting from the unique exit codes. Thoughts?

Reply:
@coreyhulen Might remember why we did this.

Reply:
I think it was sometimes like...you can miss the logging message because the app will exit before logging to the file completes.  This code will make it onto the console logs.

Reply:
but yeah, not sure if it's really adding value.